docs/pages/guides/schema-validation.mdx

```diff
@@ -0,0 +1,160 @@
+---
+meta:
+  title: "Schema validation"
+  parentTitle: "Guides"
+  description:
+    "Learn how to use schemas to ensure data integrity in your application"
+---
+
+<Banner title="Private beta">
+
+Schema validation is currently in private beta. If you would like to try it,
+please fill out the following survey to join our
+[waiting list](https://docs.google.com/forms/d/197IrpiyeqlJo9xOCOfzwdcwBIk6nisk9pToJe3st3Ng/viewform).
+
+</Banner>
+
+## Why schema validation?
+
+Schema validation is essential for future-proofing your application and ensuring
+that implementing new features or data structures into your app will keep the
+integration you have established with Liveblocks.
+
+We currently mitigate the risk of introducing client-side errors by allowing you
+to type your storage in `liveblocks.config.ts`. Still, we want to go one step
+further to help you protect your application by attaching a schema to a room
+that rejects invalid modifications.
+
+By using schema validation, you will be able to:
+
+- Trust any incoming storage modifications
+- Synchronize our storage with your database safely
+- Write migration scripts for your room storage more easily
+
+## How schema validation works
+
+The primary purpose of schema validation is to prevent corrupted data from being
+loaded from the server. To add schema validation to your application, take the
+following steps:
+
+1. Define the shape of your storage with a schema definition (similar to how you
+   would describe the storage type in TypeScript)
+2. Attach your schema to a room
+
+By default, a room storage accepts any modifications coming from the client. But
+once you attach a schema to a room, Liveblocks will reject any modifications to
+the storage that do not match the schema you provided. Situations like this
+should only happen in development mode, and the developer is responsible for
+fixing them.
+
+## Create a schema
+
+Creating a schema via the dashboard is straightforward. Simply navigate to the
+“Schemas” page in the project you want to add the schema to, and click on the
+“Create schema” button. Provide a name and a definition that describes the shape
+of the data you want to store in your rooms.
+
+<Figure>
+  <video autoPlay loop muted playsInline>
+    <source
+      src="/images/docs/schema-validation/create-schema-dashboard.mp4"
+      type="video/mp4"
+    />
+  </video>
+</Figure>
+
+<Banner title="Schema language specification">
+
+Take a look at our
+[language specification](/docs/pages/guides/schema-validation/language) to learn
```

Comment:
Page is missing, or link is dead

Reply:
This is on the todo to accomplish before releasing it: 
https://github.com/liveblocks/liveblocks/pull/647#issuecomment-1418317443